src/Symfony/Bundle/FrameworkBundle/Resources/config/import_maps.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+/*
+ * @author Kévin Dunglas <kevin@dunglas.fr>
+ */
+
+use Symfony\Component\ImportMaps\Command\AbstractCommand;
+use Symfony\Component\ImportMaps\Command\ExportCommand;
+use Symfony\Component\ImportMaps\Command\RemoveCommand;
+use Symfony\Component\ImportMaps\Command\RequireCommand;
+use Symfony\Component\ImportMaps\Command\UpdateCommand;
+use Symfony\Component\ImportMaps\ImportMapManager;
+
+return static function (ContainerConfigurator $container): void {
+    $container->services()
+
+        ->set(ImportMapManager::class)
+            ->args([param('kernel.project_dir').'/importmap.php', null, service('filesystem')])
```

Comment:
The `%kernel.project_dir%/config` folder may be a better candidate for this file.

Reply:
In my opinion it's more logic to have the list of JS packages in the same root directory, along with `composer.json` and usually `package.json`.

Reply:
I could be customizable in the config.